react/features/participants-pane/components/web/FooterContextMenu.tsx

```diff
@@ -87,7 +88,7 @@ type Props = {
 
 export const FooterContextMenu = ({ isOpen, onDrawerClose, onMouseLeave }: Props) => {
     const dispatch = useDispatch();
-    const isModerationSupported = useSelector(isAvModerationSupported);
+    const isModerationSupported = useSelector((state: IState) => isAvModerationSupported()(state));
```

Comment:
Random observation: this pattern of having a function than returns a function which needs the state passed is very odd.

Reply:
I agree. We need to change it. Here I'm just fixing the error

Reply:
👍